src/pages/Birthday.jsx

```diff
@@ -0,0 +1,10 @@
+import { useNavigate } from "react-router";
```

Comment:
I really like the separation between the `/pages`, which appear to be concerned with the entire experience at a given URL and the components, which could be re-used in any page.
Great separation of concerns!